src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -1,19 +1,111 @@
 package org.opentripplanner.middleware.controllers.api;
 
+import com.beerboy.ss.ApiEndpoint;
+import com.twilio.rest.verify.v2.service.Verification;
+import com.twilio.rest.verify.v2.service.VerificationCheck;
+import org.eclipse.jetty.http.HttpStatus;
 import org.opentripplanner.middleware.auth.Auth0UserProfile;
 import org.opentripplanner.middleware.models.OtpUser;
 import org.opentripplanner.middleware.persistence.Persistence;
+import org.opentripplanner.middleware.utils.JsonUtils;
+import org.opentripplanner.middleware.utils.NotificationUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+
+import static com.beerboy.ss.descriptor.MethodDescriptor.path;
+import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
 
 /**
- * Implementation of the {@link AbstractUserController} for {@link OtpUser}.
+ * Implementation of the {@link AbstractUserController} for {@link OtpUser}. This controller also contains
+ * OtpUser-specific endpoints for verifying the user's phone number by SMS.
  */
 public class OtpUserController extends AbstractUserController<OtpUser> {
+    private static final Logger LOG = LoggerFactory.getLogger(OtpUserController.class);
+
+    private static final String CODE_PARAM = "code";
+    protected static final String CODE_PATH = "/:" + CODE_PARAM;
+    private static final String VERIFY_ROUTE = "/verify_sms";
+
     public OtpUserController(String apiPrefix) {
         super(apiPrefix, Persistence.otpUsers, "secure/user");
     }
 
+    @Override
+    protected void buildEndpoint(ApiEndpoint baseEndpoint) {
+        LOG.info("Registering path {}.", ROOT_ROUTE + "verify");
+
+        // Add the api key route BEFORE the regular CRUD methods
+        ApiEndpoint modifiedEndpoint = baseEndpoint
+            .get(path(ROOT_ROUTE + ID_PATH + VERIFY_ROUTE)
+                    .withDescription("Request an SMS verification to be sent to an OtpUser's phone number.")
+                    .withPathParam().withName(ID_PARAM).withDescription("The id of the OtpUser.").and()
```

Comment:
Just revisiting this code prior to UI implementation... As written, we won't be able to use this process for new signups, because the `OtpUser` entry in Mongo has not been created yet... Let me think of a workaround.

Reply:
Per discussion, no changes are needed here. The UI will create the user entry upon the user accepting the terms.